src/System.Private.ServiceModel/src/System/ServiceModel/Channels/RTSocketConnection.cs

```diff
@@ -190,7 +190,7 @@ protected override void ShutdownCore(TimeSpan timeout)
             catch (ObjectDisposedException objectDisposedException)
             {
                 Exception exceptionToThrow = ConvertObjectDisposedException(objectDisposedException, TransferOperation.Undefined);
-                if (object.ReferenceEquals(exceptionToThrow, objectDisposedException))
```

Comment:
We should not use resharper to fix up the code.  The way we have done this in the past (per CoreFx recommendations) is https://github.com/dotnet/codeformatter/releases .  It was invented to make all the dotnet code follow the same conventions.  It might cause other changes in other files, which we wouldn't want in this PR.